lessons/en/advanced/otp_supervisors.md

```diff
@@ -1,5 +1,5 @@
 %{
-  version: "1.1.2",
+  version: "1.1.3",
```

Comment:
Since this is more than a typo let's bump the minor version so we can apply this change to other languages

```suggestion
  version: "1.2.0",
```

Reply:
Sorry, I might have accidentally dismissed your review.

Reply:
I think i have fixed it, thanks.